### PR TITLE
Default tolerance for Double and Float is now epsilon.

### PR DESCRIPTION
**Pull request by emilwestergren**
@sebastianbaginski @marcusnaslund 

Any objections?

### CODE REVIEW

File: test/sdk/MathTest.ooc [hunk 1]
```diff
@@ -120,8 +120,8 @@ MathTest: class extends Fixture {
 			expect(13.0 clamp(9.0, 11.0), is equal to(11.0) within(doubleTolerance))
 			expect((-2.0) clamp(-1.9, 5.0), is equal to(-1.9) within(doubleTolerance)) // (-2.0) with parentheses because of bug in rock
 
-			expect(1.999 equals(2.0), is false)
-			expect(1.99999 equals(2.0), is true)
+			expect(1.999999999999999 equals(2.0), is false)
+			expect(1.9999999999999999 equals(2.0), is true)
```
Comment (marcusnaslund) [inline test/sdk/MathTest.ooc]: Did you try as many nines as possible until it failed?

File: source/sdk/math.ooc [hunk 2]
```diff
@@ -78,12 +78,13 @@ extend Int {
 	minimum: static func (x, y: This) -> This { x < y ? x : y }
 }
 
+DBL_EPSILON: extern Double
```
Comment (marcusnaslund) [inline source/sdk/math.ooc]: Put this at the top with the other one

File: test/sdk/MathTest.ooc (hunk 1 above)
Comment (emilwestergren) [inline test/sdk/MathTest.ooc]: yes

**Comment by marcusnaslund** [comment]
@sebastianbaginski - this one's yours to peer review now.

File: source/sdk/math.ooc [hunk 3]
```diff
@@ -2,6 +2,7 @@ include math
 use math
 
 FLT_EPSILON: extern Float
+DBL_EPSILON: extern Double
```
Comment (sebastianbaginski) [inline source/sdk/math.ooc]: Do we want to have a nice name here, like `DoubleEpsilon: extern (DBL_EPSILON) Double` ? @marcusnaslund ?

Comment (marcusnaslund) [inline source/sdk/math.ooc]: I guess we could, but the way they're meant to be accessed is through the `epsilon` property of `Double` or `Float`.

**Pull request merged**